interface/modules/custom_modules/oe-module-ehi-exporter/templates/oe-module-ehi-exporter/ehi-exporter.html.twig

```diff
@@ -0,0 +1,116 @@
+<html>
+<head>
+    <title>{{ "Electronic Health Information Export"|xlt }}</title>
+    {{ setupHeader() }}
+</head>
+    <body class="p-3">
+    {% if result %}
+        {% if result.getStatus() == 'completed' %}
+            <h1>{{ "Exported Electronic Health Information (EHI) Zip Files"|xlt }}</h1>
+            {% for task in result.getJobTasks() %}
+                <h4><a download href="{{ task.exportedResult.downloadLink|attr }}">{{ task.document.get_name()|text }} - {{ "Download"|xlt }}</a></h4>
+                <h6>{{ task.document.get_hash_algo_title()|text }} <small>{{ task.document.get_hash()|text }}</small></h6>
+
+            <details class="mb-5">
+                <summary>
+                    {{ "Expand for Export Zip Details"|xlt }}
+                </summary>
+                {% if task.exportedResult.exportedTables %}
+                <p>{{ "Records Exported"|xlt }}</p>
+                <ol>
+                    {% for tableResult in task.exportedResult.exportedTables %}
+                    <li>{{ tableResult.tableName|text}}: {{tableResult.count|text}}</li>
+                    {% endfor %}
+                </ol>
+                {% endif %}
+
+                {% if result.include_patient_documents %}
+                <p>{{ "Documents Exported"|xlt }}: {{ task.exportedResult.exportedDocumentCount|xlt }}</p>
```

Comment:
the task.exportedResult.exportedDocumentCount should use |text and not |xlt

Reply:
Should be resolved now.